src/main/java/org/opentripplanner/routing/alternativelegs/AlternativeLegs.java

```diff
@@ -45,14 +45,25 @@ public static List<ScheduledTransitLeg> getAlternativeLegs(
     TransitService transitService,
     boolean searchBackward,
     AlternativeLegsFilter filter
+  ) {
+    return getAlternativeLegs(leg, numberLegs, transitService, searchBackward, filter, false);
+  }
+
+  public static List<ScheduledTransitLeg> getAlternativeLegs(
```

Comment:
I wonder if we should put javadoc here to explain what some of these arguments do.